### PR TITLE
KTX2 loader: sets the hasAlpha property of the texture

### PR DESCRIPTION
**Pull request by Popov72**
Automatically sets the `Texture.hasAlpha` property if the loaded file has an alpha channel.

Also, configure the local environment to point to the local ktx2Decoder files.

### CODE REVIEW

**Comment by sebavan** [comment]
Would there be a way to not move HasAlpha on the internal texture ? This was originally a user set flag to turn alpha test on and not meant to be filled by what is actually in the texture. The texture type should provide us with the underlying data ?

**Comment by Popov72** [comment]
All the underlying loading code deals only with internal textures, so I don't know how to bubble this data up until we have a Texture available...

Note that having `hasAlpha` on the internal texture is not a breaking change and the user can still update the value as they see fit (meaning setting it to true even if the texture has no alpha channel and setting it to false even if the texture has an alpha channel). It's the same change that I already did for `gammaSpace`, which was a `BaseTexture` property at start and which was moved to `InternalTexture` but without breaking things.

**Comment by sebavan** [comment]
Ok so all good to me :-) I ll let @bghgary review and merge it in

File: Playground/index-local.html [hunk 1]
```diff
@@ -55,6 +55,12 @@
         <div id="host-element">
         </div>
         <script>
+            function GetAbsoluteUrl(url) {
+                const a = document.createElement("a");
+                a.href = url;
+                return a.href;
+            }
+
```
Comment (bghgary) [inline Playground/index-local.html:63]: I know this is copied this from elsewhere, but I just noticed that this won't work in Babylon Native. We will need to do something different here and elsewhere.

File: tests/validation/config.json [hunk 2]
```diff
@@ -921,7 +921,8 @@
         },
         {
             "title": "KTX2 decoder test",
-            "playgroundId": "#EIJH8L#1",
+            "playgroundId": "#EIJH8L#26",
+            "renderCount": 5,
```
Comment (bghgary) [inline tests/validation/config.json:925]: Why do you need a render count?

Comment (Popov72) [reply tests/validation/config.json]: I don't remember why, it works without it - I have removed it.

**Comment by sebavan** [comment]
I guess it is now all good to merge @Popov72 ???

**Comment by Popov72** [comment]
Ok for me!

**Pull request merged**